src/vast_tracker.js

```diff
@@ -37,6 +37,8 @@ export class VASTTracker extends EventEmitter {
     this.impressed = false;
     this.skippable = false;
     this.trackingEvents = {};
+    this.lastPercentage = 0; //We need to keep the last percentage of the tracker in order to
+    //calculate to trigger the events when the VAST duration is short
```

Comment:
For better readability, the comment should be moved to the top of the corresponding line.
```suggestion
    this.trackingEvents = {};
    // We need to keep the last percentage of the tracker in order to
    // calculate to trigger the events when the VAST duration is short
    this.lastPercentage = 0; 
```

Reply:
Doesn't look resolved to me 😅